test/k8sT/Services.go

```diff
@@ -672,18 +672,22 @@ var _ = Describe("K8sServicesTest", func() {
 			), "Failed to account for IPv4 fragments to %s (out)", dstIP)
 		}
 
-		getIPv4Andv6AddrForIface := func(nodeName, iface string) (string, string) {
+		getIPv4AddrForIface := func(nodeName, iface string) string {
 			cmd := fmt.Sprintf("ip -4 -o a s dev %s scope global | awk '{print $4}' | cut -d/ -f1", iface)
 			res := kubectl.ExecInHostNetNS(context.TODO(), nodeName, cmd)
 			res.ExpectSuccess(cmd)
 			ipv4 := strings.Trim(res.Stdout(), "\n")
 
-			cmd = fmt.Sprintf("ip -6 -o a s dev %s scope global | awk '{print $4}' | cut -d/ -f1", iface)
-			res = kubectl.ExecInHostNetNS(context.TODO(), nodeName, cmd)
+			return ipv4
+		}
+
+		getIPv6AddrForIface := func(nodeName, iface string) string {
+			cmd := fmt.Sprintf("ip -6 -o a s dev %s scope global | awk '{print $4}' | cut -d/ -f1", iface)
```

Comment:
Nit: I find it helpful to put the long names of args, etc in scripts / tests so that it is clear what is happening without needing to remember all the aliases or having to look them up.

Reply:
Do you mean using `ip -family inet6 -oneline address show dev wlo1 scope global | awk '{print $4}' | cut -d/ -f1` instead?

Reply:
Haha, that's quite long isn't now. I think you can omit the `family` and the IPv6 stuff. I was mostly interested in the actual subcommands to `ip`. In this case they are `address` and `show`.

```
ip -6 -oneline addr show dev ...
``` 

Looks good to me ^